edge-modules/azure-monitor/src/MetricsScraper.cs

```diff
@@ -40,15 +40,15 @@ public MetricsScraper(IList<string> endpoints, ISystemTime systemTime = null)
         /// <summary>
         /// Scrapes metrics from all endpoints
         /// </summary>
-        public Task<IEnumerable<Metric>> ScrapeEndpointsAsync(CancellationToken cancellationToken)
+        public Task<IEnumerable<Tuple<Metric, String>>> ScrapeEndpointsAsync(CancellationToken cancellationToken)
```

Comment:
Would it be better to use this syntax to avoid using item1 and item2 ?

```public Task<IEnumerable<(Metric metric, String tag)>> ScrapeEndpointsAsync(CancellationToken cancellationToken)```

Reply:
This is definitely the right thing to do!